lib/heroku/helpers.rb

```diff
@@ -283,7 +283,7 @@ def error(message, report=false, opts={})
       rollbar_id = Rollbar.error(message) if report
       $stderr.puts("Error ID: #{rollbar_id}") if rollbar_id
       request_id = opts[:error] && opts[:error].response ? opts[:error].response.headers['Request-Id'] : nil
-      $stderr.puts "Request ID: #{request_id}" if request_id
+      $stderr.puts "Request ID: #{request_id}" if report && request_id
```

Comment:
ah you're right. This was way too noisy, we often have the API respond with errors about things like an app name already existing. It's too noisy to have the request id on all of that. I'm going to roll back both changes.